### ISSUE TITLE
File Entity: Use New UIKit Icon for "Open Lock" When Available

### ISSUE THREAD

**Issue by mistryrn** [icgc-argo/platform-ui#2136]
## Detailed Description
To prevent forking changes between the previous uikit housed within platform-ui and the new separated uikit repo, the adding of the new "Open Lock" icon for https://github.com/icgc-argo/platform-ui/issues/2090 to uikit was put on hold until the new UIKit is ready.

Once the new UIKit is ready, the "Open Lock" icon will be added there and can be implemented on the File Entity page in platform-ui.

Related UIKit issue: https://github.com/icgc-argo/uikit/issues/26
